src/afpacket_plugin/afpacket_collector.cpp

```diff
@@ -210,6 +210,11 @@ int setup_socket(std::string interface_name, int fanout_group_id) {
 
     mapped_buffer = (uint8_t*)mmap(NULL, req.tp_block_size * req.tp_block_nr, PROT_READ | PROT_WRITE, MAP_SHARED | MAP_LOCKED, packet_socket, 0);
 
+    // Musl libc (Alpine Linux) doesn't seem to like MAP_LOCKED, retry without it
```

Comment:
Could you share error in this case, please? I do not think that I understand AF_PACKET internals very well and could predict performance impact in this case.